widget/menu.go

```diff
@@ -18,6 +18,66 @@ type Menu struct {
 	OnDismiss   func()
 	activeItem  *menuItem
 	customSized bool
+	index       int
+}
+
+func (m *Menu) Defocus() {
+	for _,o := range m.Items {
+		if mi, ok := o.(*menuItem); ok {
+			mi.FocusLost()
+		}
+	}
+}
+
+func (m *Menu) Do() {
+	if m.activeItem!=nil {
+		m.activeItem.child.Do()
+		return
+	}
+	if o, ok := m.Items[m.index].(*menuItem); ok {
+		o.MouseIn(nil)
+		o.Tapped(nil)
+	}
+}
+
+func (m *Menu) Up() {
```

Comment:
Again, naming unclear and not a sensible public API

Reply:
See above. I have now tried to improve naming in next version coming soon.